specification/web/resource-manager/Microsoft.Web/stable/2024-04-01/ResourceProvider.json

```diff
@@ -545,6 +545,98 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/locations/{location}/virtualNetworkIntegrations": {
```

Comment:
>virtualNetworkIntegrations [](http://example.com/codeflow?start=82&length=26)

Why is this a POST call?

Reply:
Because we need to pass in a body that contains the subnet resource id. This body is required for linkedAccessCheck in ARM.

Reply:
Has this been discussed in API Review Modelling office hours? Are you sure there is no other way to model this as a GET request? The description makes it clear to the client that this is a read operation.

Reply:
@ms-bogdan-rotaru if you know of another way or examples from other RPs, we are happy to look at it, but to the best of our knowledge, this is the only way to have ARM Linked Access Check on a required parameter in the call.